BusinessTampereTrafficMonitoring/traffic_lights/signal_group.py

```diff
@@ -3,6 +3,16 @@
 
 class SignalGroup:
     def __init__(self, device: str, name: str, timestamp: str, status_code: str):
+        """
+        Constructor to initialize the signal group
+        # Parameter:
+        # argument1: device id (String)
+        # argument2: device name (String)
```

Comment:
Not a device name. This is the name of the signal group (for example "A", or "RV1").